spec/dmv_spec.rb

```diff
@@ -37,6 +37,7 @@
       @dmv.add_facility(@facility_2)
       @dmv.add_facility(@facility_3)
 
+      p @dmv.facilities_offering_service('Road Test')
```

Comment:
You'll want to remove `p` and `puts` from your tests in the future when you're all wrapped up with a project. :)